app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    puts comment_params
+    @comment = Comment.new comment_params
+    puts 'VALID?: ', @comment.valid?
+    @comment.save if @comment.valid?
+    redirect_to post_path(@comment.post)
```

Comment:
- [ ] OPTIONAL: Please try to keep your codebase clean and do not merge lines of code that is not relevant to project. I strongly suggest you to delete puts before merging to develop branch 🙏🏻